eng/Baseline.Designer.props

```diff
@@ -576,7 +576,7 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.SignalR.Protocols.MessagePack' AND '$(TargetFramework)' == 'netstandard2.0' ">
     <BaselinePackageReference Include="Microsoft.AspNetCore.SignalR.Common" Version="[3.1.0, )" />
-    <BaselinePackageReference Include="MessagePack" Version="[1.7.3.7, )" />
+    <BaselinePackageReference Include="MessagePack" Version="[2.0.323, )" />
```

Comment:
Revert changes to this file

Reply:
done,

Just found out, https://github.com/aspnet/AspNetCore/blob/master/docs/ReferenceResolution.md#important-files 
I'll try to address review here and read it carefully